spring-cloud-gcp-samples/pom.xml

```diff
@@ -113,6 +113,25 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>integration-tests</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<systemPropertyVariables>
+								<gcs-resource-test-bucket>gcp-storage-resource-bucket-sample</gcs-resource-test-bucket>
```

Comment:
This seems OK, but it's important to realize that it's only possible to use this profile with the `spring-cloud-gcp-ci` project. Since bucket names are globally unique, you can't just create one with the same name in another project. So, maybe it would be more clear to name it `spring-cloud-gcp-ci-it` or something like that.

Reply:
Done.